op-batcher/batcher/driver.go

```diff
@@ -647,7 +660,7 @@ func (l *BatchSubmitter) clearState(ctx context.Context) {
 			return false
 		} else {
 			l.Log.Info("Clearing state with safe L1 origin", "origin", l1SafeOrigin)
-			l.state.Clear(l1SafeOrigin)
+			l.channelMgr.Clear(l1SafeOrigin)
```

Comment:
don't we need to guard this call with the mutex?

Reply:
This function is only called:
* by StartBatchSubmitting _before_ any goroutines are launched and 
* by syncAndPublish, which is already holding the lock

So we shouldn't add a guard here, because it is unecessary and would cause a deadlock.